File: app/src/main/java/com/example/mobdev21_night_at_the_museum/model/ItemDetail.java
package com.example.mobdev21_night_at_the_museum.model;

public class ItemDetail {
    private String title;
    private String description;

    public ItemDetail() {
    }

    public ItemDetail(String title, String description) {
        this.title = title;
        this.description = description;
    }

    public void mapFrom(String string) {
        String[] split = string.split(":");
        if (split.length == 2) {
            title = split[0] + ": ";
            description = split[1].trim();
        }
    }

    @Override
    public String toString() {
        return "<b>" + title + "</b>: " + description + "<br>";
    }
}

File: app/src/main/java/com/example/mobdev21_night_at_the_museum/domain/model/ItemDetail.kt
package com.example.mobdev21_night_at_the_museum.domain.model

import kotlinx.parcelize.Parcelize

@Parcelize
data class ItemDetail(

    var title: String? = null,

    var description: String? = null

) : MuseumModel() {

    // from Title: Jason to title = "Title:" and description = "Jason"
    fun mapFrom(string: String) {
        val split = string.split(":")
        title = split[0] + ":"
        description = split[1].trim()
    }
}
